src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,4 @@
+*REMOVED*Microsoft.CodeAnalysis.Operations.ICollectionElementInitializerOperation.Arguments.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.IOperation>
```

Comment:
>OVED [](start = 4, length = 4)

Removal of a shipped API generally requires compat council approval. Is there a different bar here?

Reply:
API is not getting removed, no longer relevant.

---
In reply to: [187511037](https://github.com/dotnet/roslyn/pull/26781#discussion_r187511037) [](ancestors = 187511037)